.eslintrc.js

```diff
@@ -22,8 +22,11 @@ module.exports = {
     "prettier/prettier": ["error"],
     "no-use-before-define": ["error", { functions: false, classes: true }],
     "no-shadow": "off",
+    "no-console": 0,
```

Comment:
I'm not sure if the desired outcome of pointing out all of those `eslint-disable` lines was to have you disable the rules globally.